model/shared/src/main/scala/explore/model/boopickle/CatalogPickler.scala

```diff
@@ -105,14 +105,14 @@ trait CatalogPicklers extends CommonPicklers {
       .addConcreteType[AgsAnalysis.NoMagnitudeForBand]
       .addConcreteType[AgsAnalysis.Usable]
 
-  given Pickler[ObjectTracking.ConstantTracking] = generatePickler
-
   given Pickler[ObjectTracking.SiderealObjectTracking] = generatePickler
 
+  given Pickler[ObjectTracking.SiderealAsterismTracking] = generatePickler
+
   given Pickler[ObjectTracking] =
     compositePickler[ObjectTracking]
-      .addConcreteType[ObjectTracking.ConstantTracking]
       .addConcreteType[ObjectTracking.SiderealObjectTracking]
+      .addConcreteType[ObjectTracking.SiderealAsterismTracking]
```

Comment:
We need to invalidate the catalog cache with this change

Reply:
Does the cache rely on the `ObjectTracking` or on the coordinates based on the `ObjectTracking`?

Reply:
As discussed we don't really need to invalidate